CONTRIBUTING.md

```diff
@@ -11,6 +11,10 @@
   - [Breaking Changes](#breaking-changes)
```

Comment:
I rather have `TESTING.md` just because is a nice practice and I know I needed to go to that file to file what I needed and didn't need to scan a file with many lines.

So my two cents is having multiple files with smaller lines to read through.

Reply:
I am of that opinion too. I like to have the separation so it makes for easy to consume chunks. @orlandohohmeier pointed out though that the separation we have right now is a little confusing, and the links not really good. I think it is better to have a large text, that content that is intuitive to find.

What if I leave the TESTING.md file (because in this case, it is indeed a standard) and make it point to the CONTRIBUTING document?